webapp/src/pages/Settings.tsx

```diff
@@ -2,27 +2,58 @@ import {
   Box,
   Button,
   Checkbox,
-  Container,
-  FormControl,
+  Divider,
   FormControlLabel,
-  FormGroup,
   FormHelperText,
+  InputBaseComponentProps,
+  Paper,
+  TextField,
+  Typography,
 } from "@mui/material";
+import _ from "lodash";
 import React from "react";
+import AccordionLayout from "components/AccordionLayout";
 import { useParams } from "react-router-dom";
 import { getConfigEndpoint, updateConfigEndpoint } from "services/api";
-import { AzimuthConfig } from "types/api";
+import { AzimuthConfig, PipelineDefinition } from "types/api";
+
+type stepper = { [key: string]: InputBaseComponentProps };
+const STEPPER: stepper = {
```

Comment:
Types should be capitalized. Anyway, I think we don't need to save the `type`, we can use it directly:
```suggestion
const STEPPER: Record<string, InputBaseComponentProps> = {
```